rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -670,6 +673,47 @@ public Topology getTopology(@HeaderParam("sessionid") String sessionId) throws N
         return rm.getSupportedNodeSourceInfrastructures();
     }
 
+    @Override
+    @GET
+    @GZIP
+    @Path("infrastructuresmapping")
```

Comment:
I think that `/infrastructures/mapping` or `/infrastructures/policies` would be more relevant. It is better to keep the key "infrastructures".

Reply:
ok